### ISSUE TITLE
Content dissapears on dimension change.

### ISSUE THREAD

**Issue by Neb2566** [TheRealZeher/DimensionalPockets-II#25]
Steps to reproduce,

Place Pocket on overworld, place item inside, pick up, change to aroma mining world, content gone, take it back to overworld, items are back.

So seems content dissapears on dimension change, but comes back if you go back to original dimension.

**Comment by TheRealZeher** [comment]
At present, the pocket block does not retain the information required to link back to the pocket it first creates. This causes a new pocket to be generated and spawned whenever a pocket block is placed down. When a pocket block is picked up then placed down in a different place, it acts the same as placing a new pocket block. When a pocket block is placed in the same position as a removed pocket block, that new pocket block will link back to the original. This is due to the fact that i haven't yet worked out how to attach the pocket data to the ItemBlock and then call it back whenever the pocket is re-placed. I am looking into this issue urgently, as i know that it is a key function of the original mod. 

TLDR; Picking up and placing a pocket block in a different location does not link to original pocket created. I am aware of this issue.

**Comment by Neb2566** [comment]
Ah, thought it was due to dimension change. Might want to talk to Davenonymous, creator or Compact Machines, afaik he has solved this issue. Tho I like your mod better due to crafting mechanics : )

**Comment by TheRealZeher** [comment]
I have a few ideas that I might be able to carry over from my other mod. I also have yet to refer back to the original code for this specific part, although from what I remember, because minecraft has changed so much the original way of doing it no longer works. This is the most urgent issue so is top priority for the mod at present. I hope to get it working asap.

**Comment by TheRealZeher** [comment]
Issue fixed for version 2.0.6. NBT Data added so pocket block links wherever it is placed.

**Comment by Neb2566** [comment]
Downloaded latest versions of pocket dim and library, still same bug.

**Comment by TheRealZeher** [comment]
Are you using the wrench to pick up the pocket block? Shift-right clicking will pick the block up the correct way.

**Comment by Neb2566** [comment]
Ah, my bad, did not use the wrench no.

**Comment by Neb2566** [comment]
Had a chance to test it out now, and it works perfectly, awesome work!

**Comment by TheRealZeher** [comment]
Awesome, glad it works :P
